pkg/nsx/services/ipblocksinfo/ipblocksinfo.go

```diff
@@ -395,8 +433,47 @@ func (s *IPBlocksInfoService) getIPBlockCIDRsFromStore(pathSet sets.Set[string],
 		if ipblock.Cidr == nil { //nolint:staticcheck //ipblock.Cidr is deprecated
```

Comment:
Can we combine getIPBlockCIDRFromStore and getIPBlockCIDRsFromStore together?
if ipblock.Cidrs != nil: get from cidrs
else if ipblock.Cidr != nil: get from cidr
else return nil, NoCIDRsFoundError

Reply:
I merged these two functions. but not return NoCIDRsFoundError since ipblocks may have only ranges